skaha/src/main/java/org/opencadc/skaha/session/SessionJobBuilder.java

```diff
@@ -1,14 +1,7 @@
 package org.opencadc.skaha.session;
 
-import io.kubernetes.client.openapi.models.V1Affinity;
-import io.kubernetes.client.openapi.models.V1Job;
-import io.kubernetes.client.openapi.models.V1JobSpec;
-import io.kubernetes.client.openapi.models.V1NodeAffinity;
-import io.kubernetes.client.openapi.models.V1NodeSelector;
-import io.kubernetes.client.openapi.models.V1NodeSelectorRequirement;
-import io.kubernetes.client.openapi.models.V1NodeSelectorTerm;
-import io.kubernetes.client.openapi.models.V1PodSpec;
-import io.kubernetes.client.openapi.models.V1PreferredSchedulingTerm;
+import ca.nrc.cadc.util.StringUtil;
+import io.kubernetes.client.openapi.models.*;
```

Comment:
Im split between wildcard import statements and explicit individual statements for readability.